docs/design/baremetal/networking-infrastructure.md

```diff
@@ -69,6 +70,11 @@ https://github.com/openshift/machine-config-operator/pull/795
 See [here](FIXME: link to a README in MCO) for more detailed information about
 the `haproxy` configuration.
 
+For clients inside the cluster, the Kubernetes apiserver is accessible via the
```

Comment:
This information is technically not baremetal specific and it's general kubernetes.. I don't think this is required.

Reply:
@abhinavdahiy - that's correct, will delete this part